content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,31 @@
+---
+title: IMDSv2 Enablement by Default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Enablement by Default – What You Need to Know
```

Comment:
```suggestion
```

You actually don't need this title here at all, the title is already displayed in line 2. also, this is a documentation page, so "what you need to know" is implied :)